src/http/url.cc

```diff
@@ -67,9 +67,8 @@ inline char char_to_hex(unsigned char val) {
     return "0123456789ABCDEF"[val];
 }
 
-}
-
-bool url_decode(const std::string_view& in, sstring& out) {
+template<bool ReplacePlus>
+bool decode(const std::string_view& in, sstring& out) {
```

Comment:
I think a boolean parameter would have sufficed, but we can let the optimiser figure it out.

Reply:
I've simplified this into a regular boolean parameter now